docs/getting-started.md

```diff
@@ -55,7 +55,7 @@ latest=$(basename $(curl -fsSLI -o /dev/null -w  %{url_effective} https://github
 ```
 
 ```wrap
-curl -LOJ https://github.com/firecracker-microvm/firecracker/releases/download/${latest}/firecracker-${latest}-$(uname -m)
+curl -L https://github.com/firecracker-microvm/firecracker/releases/download/${latest}/firecracker-${latest}-$(uname -m).tgz | tar -xz
```

Comment:
I receive this error when trying to run this:
```
gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
```

Why not
```
curl -LO https://github.com/firecracker-microvm/firecracker/releases/download/${latest}/firecracker-${latest}-$(uname -m).tgz
```
?

Reply:
> I receive this error when trying to run this:
> 
> ```
> gzip: stdin: not in gzip format
> tar: Child returned status 1
> tar: Error is not recoverable: exiting now
> ```

I suspect $latest is not defined in your case so, github server may have responded with "Not Found" string and tar tried to parse it as a gzip format. can you check what is the output of `echo https://github.com/firecracker-microvm/firecracker/releases/download/${latest}/firecracker-${latest}-$(uname -m).tgz` 🙂

> Why not
> 
> ```
> curl -LO https://github.com/firecracker-microvm/firecracker/releases/download/${latest}/firecracker-${latest}-$(uname -m).tgz
> ```

Nothing much. Just tried to download and untar in a single line. 

If we use `-O` then the file will be saved to disk (with the name in the header) and then we have to untar the downloaded tar file. It takes two steps.

Reply:
I think that this looks good. As @KarthikNedunchezhiyan pointed out, the `latest` variable needs to be defined first.

@dianpopa can you confirm that this works?

Reply:
Indeed latest was not set. It works now.